ReferenceTests/src/tests/text.jl

```diff
@@ -298,7 +298,7 @@ end
 
     Makie.step!(st)
     ## change lengths
-    textnode.val = push!(textnode[], L"\int_0^5x^2+2ab")
```

Comment:
With the changes in the pr this doesn't work anymore for CairoMakie, but updating the nodes in either order normally works

Reply:
what does that mean?

Reply:
I think I made the `textnode.val = ...` -> `textnode[] = ...` explicitly for CairoMakie tests to pass

Reply:
ah that way around...